index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Sharp:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"/>
+    <!-- <link rel="stylesheet" href="https://kit.fontawesome.com/058c29e44e.css" crossorigin="anonymous"> -->
+</head>
+<body>
+    <header class="header">
+        <div class="logo-container">
+          <i class="material-symbols-rounded">menu</i>
+            <div class="logo">
+                <div class="logo-img">
+                    <img src="images/youtube-logo-2431.png" alt="">
+                </div>
+                <h1 id="logo-txt">YouTube</h1>
+            </div>       
+        </div>
+
+        <div class="search-container">
+            <div class=search-and-mic>
+              <form action="">
+                <input type="text" placeholder="Search">
+              </form>
+              <button class = "search"><i class="material-symbols-outlined">search</i></button>
+            </div>
+            <div class = "mic"><i class="material-symbols-outlined">mic</i></div>    
+        </div>
+
+        <div class="profile-container">
+            <!--<i class = "ri-video-add-line"></i>-->
```

Comment:
Remove unused code to keep code clean.

Reply:
Github inline commenting was used very well here as well 💯. However, as I mentioned in the previous comments, it is always a good thing to make your review comments as friendly suggestions and explain why you suggest something, being as detailed and providing as much context as you can. 👍